src/program.js

```diff
@@ -113,6 +120,11 @@ export class Program {
         throw new UsageError(`Unknown command: ${cmd}`);
       }
       await runCommand(argv);
+
+      checkForUpdates ({
```

Comment:
This needs to start before the command starts, at least it does for the `run` command. I'm not totally sure why, I think it's because of how `run` exits when Firefox quits. I think it's safe to move it up because the updater runs unobtrusively in an async child process.

````diff
diff --git a/src/program.js b/src/program.js
index 1f49966..33b34da 100644
--- a/src/program.js
+++ b/src/program.js
@@ -119,12 +119,13 @@ export class Program {
       if (!runCommand) {
         throw new UsageError(`Unknown command: ${cmd}`);
       }
-      await runCommand(argv);
 
       checkForUpdates ({
         version: getVersion(absolutePackageDir),
       });
 
+      await runCommand(argv);
+
     } catch (error) {
       const prefix = cmd ? `${cmd}: ` : '';
       if (!(error instanceof UsageError) || argv.verbose) {
````

It looks nice!

<img width="256" alt="screenshot 2016-12-21 12 25 18" src="https://cloud.githubusercontent.com/assets/55398/21401226/0b9bc204-c779-11e6-8240-9ea5bcdeb793.png">